webapp/src/js/enketo/lib/window.js

```diff
@@ -0,0 +1,5 @@
+const getCurrentHref = () => window.location.href;
+
+module.exports = {
+  getCurrentHref,
```

Comment:
Need this to provide an abstraction point for `file-manager` unit test to allow them to mock out the href. (Unless someone has a better way to do this...)

IMHO, this is annoying, but better to just include this vs wasting a ton of time (well even more time since I already wasted a ton here) trying to find a more creative solution....